deployment/deploy-dev-app.ps1

```diff
@@ -10,15 +10,28 @@ Param(
 
     [string]
     [Parameter(mandatory=$true)]
-    $AksClusterName,
+    $AksClusterNameUpper,
```

Comment:
I wonder, at some point should we agree to either L2/3/4 or Upper/Middle/Lower from terminology perspective?

Reply:
The only reason I'm unsure is the following: when we deploy the developer version with just one layer (typically L4), then we pass in L4 parameters to both $AksClusterNameUpper and $AksClusterNameLower. This is to ensure the script works whether executing for installing both Helm charts on a single layer, or when calling the same script with installation split up between L2 and L4. 
If we agree it's not too confusing to call the L2/L4 levels in the script parameter suffixes, then I suggest changing it consequently in all scripts with this. Lower becomes L2 and Upper becomes L4.